src/test/java/core/basesyntax/StoreServiceTest.java

```diff
@@ -0,0 +1,174 @@
+package core.basesyntax;
+
+import core.basesyntax.items.Storage;
+import core.basesyntax.services.DataAggregator;
+import core.basesyntax.services.LineParser;
+import core.basesyntax.services.Manager;
+import core.basesyntax.services.TextPrinter;
+import core.basesyntax.services.TextReader;
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class StoreServiceTest {
+    public final static String FIRST_PATH = "src/test/resources/test_1.csv";
+    public final static String SECOND_PATH = "src/test/resources/test_2.csv";
+    public final static String THIRD_PATH = "src/test/resources/test_3.csv";
+    public final static String FOURTH_PATH = "src/test/resources/test_4.csv";
+    public final static String FIFTH_PATH = "src/test/resources/test_5.csv";
+    public final static String SIXTH_PATH = "src/test/resources/test_6.csv";
+    public final static String FIRST_INPUT_DATA = "s,banana,100,2020-10-17";
+    public final static String WRONG_ARGUMENT_AMOUNT_TO_PARSE_DATA = "asdasdasd";
+    public final static String WRONG_QUANTITY_ARGUMENT_TO_PARSE_DATA = "s,banana,1s00r,2020-10-17";
+    public final static String WRONG_DATE_ARGUMENT_TO_PARSE_DATA = "s,banana,100,20204-103-177";
+
+    @Test
+    public void textReaderTest() {
+        TextReader reader = new TextReader();
+        String actual = reader.read(FIRST_PATH).get(0);
+        Assert.assertEquals(FIRST_INPUT_DATA, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void textReaderTestFail() {
+        TextReader reader = new TextReader();
+        reader.read("asdsd").get(0);
+    }
+
+    @Test
+    public void textReaderTestWithMultipleLines() {
+        TextReader reader = new TextReader();
+        int actual = reader.read(SECOND_PATH).size();
+        Assert.assertEquals(3, actual);
+    }
+
+    @Test
+    public void lineParserTest() {
```

Comment:
method naming 
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

Reply:
sorry, what do you mean? I've separated names by _ That's only thing I found

Reply:
lineParserTest - is noun, not verb